test/cucumber/http_response.go

```diff
@@ -53,7 +53,7 @@ func init() {
 		ctx.Step(`^the response should match "([^"]*)"$`, s.theResponseShouldMatchText)
 		ctx.Step(`^I store the "([^"]*)" selection from the response as \${([^"]*)}$`, s.iStoreTheSelectionFromTheResponseAs)
 		ctx.Step(`^I store json as \${([^"]*)}:$`, s.iStoreJsonAsInput)
-		ctx.Step(`^the "([^"]*)" selection from the response should match "([^"]*)"$`, s.theSelectionFromTheResponseShouldMatch)
+		ctx.Step(`^the "(.*)" selection from the response should match "([^"]*)"$`, s.theSelectionFromTheResponseShouldMatch)
```

Comment:
This change is required to allow more complex JQ expression such as `.items[] | select(.id == "aws-sqs-source-v1alpha1") | has("schema")`  where double quotes are required by the JQ syntax but were excluded by the original regexp.

Reply:
lgtm

Reply:
@pb82 @machi1990 @ziccardi is that change ok with you ?

Reply:
Yes. It is okay with us. The kafka side does not use cucumber at the moment...